### PR TITLE
v11.4.0

### PR DESCRIPTION
**Pull request by adamraine**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,85 @@
...
+
+## New Contributors
+
+Thanks to our new contributors 👽🐷🐰🐯🐻!
+
+- avinash-sd @avinash-sd
+
+## New Audits
+
+* third-party cookies ([#15632](https://github.com/GoogleChrome/lighthouse/pull/15632))
```
Comment (brendankenny) [inline changelog.md:15]: Add a short description?

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,85 @@
...
+## Core
+
+* reintroduce resource-summary as a hidden audit ([#15597](https://github.com/GoogleChrome/lighthouse/pull/15597))
+* move transfer ratio estimate to script-helpers.js ([#15665](https://github.com/GoogleChrome/lighthouse/pull/15665))
+* set network responseHeadersText from extra info event ([#15639](https://github.com/GoogleChrome/lighthouse/pull/15639))
+* fix check for presence of all URL artifact fields ([#15648](https://github.com/GoogleChrome/lighthouse/pull/15648))
+* fix worker request expectations for M121 ([#15601](https://github.com/GoogleChrome/lighthouse/pull/15601))
+* aria-allowed-role: fix typo in learn more link ([#15633](https://github.com/GoogleChrome/lighthouse/pull/15633))
+* bf-cache: add warning and hide when using old headless ([#15577](https://github.com/GoogleChrome/lighthouse/pull/15577))
+* bf-cache: change to use binary scoring mode ([#15581](https://github.com/GoogleChrome/lighthouse/pull/15581))
```
Comment (brendankenny) [inline changelog.md:26]: Worth pulling out into a notable section?

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,85 @@
...
+* bootup-time: exclude _lighthouse-eval.js ([#15678](https://github.com/GoogleChrome/lighthouse/pull/15678))
+* byte-efficiency: replace pessimistic graph with optimistic ([#15651](https://github.com/GoogleChrome/lighthouse/pull/15651))
+* css-usage: exclude empty stylesheets ([#15679](https://github.com/GoogleChrome/lighthouse/pull/15679))
+* duplicated-javascript: exclude header size for estimating wasted bytes ([#15667](https://github.com/GoogleChrome/lighthouse/pull/15667))
+* emulation: bump chrome UA to m119 ([#15661](https://github.com/GoogleChrome/lighthouse/pull/15661))
+* entity-classification: update tldts package to icann subset ([#15660](https://github.com/GoogleChrome/lighthouse/pull/15660))
+* entity-classification: integrate public-suffix-list into LH ([#15641](https://github.com/GoogleChrome/lighthouse/pull/15641))
```
Comment (brendankenny) [inline changelog.md:33]: Notable?

File: changelog.md [hunk 4]
```diff
@@ -1,3 +1,85 @@
...
+* mpfid: add max LoAFs to debugdata ([#15684](https://github.com/GoogleChrome/lighthouse/pull/15684))
+* third-party-summary: expands the audit to include all urls ([#15611](https://github.com/GoogleChrome/lighthouse/pull/15611))
+* trace-elements: remove element score field ([#15677](https://github.com/GoogleChrome/lighthouse/pull/15677))
+* unminified-javascript: exclude header size for estimating wasted bytes ([#15670](https://github.com/GoogleChrome/lighthouse/pull/15670))
+* unused-javascript: exclude header size for estimating wasted bytes ([#15668](https://github.com/GoogleChrome/lighthouse/pull/15668))
+* uses-http2: include multiplexable assets when 1p is a known 3p origin ([#15638](https://github.com/GoogleChrome/lighthouse/pull/15638))
+
+## Report
+
+* blend impact and guidance level to sort audits ([#15669](https://github.com/GoogleChrome/lighthouse/pull/15669))
```
Comment (brendankenny) [inline changelog.md:49]: Worth highlighting? Or we can do an eventual summary of all sorting changes

**Review by brendankenny — APPROVED** [review]
LGTM!

File: changelog.md (hunk 3 above)
Comment (adamraine) [reply changelog.md:33]: Nah, it subtly affects how entities are classified

File: changelog.md (hunk 4 above)
Comment (adamraine) [reply changelog.md:49]: Nah I don't think we need to highlight every sorting change.

File: changelog.md (hunk 2 above)
Comment (adamraine) [reply changelog.md:26]: I'm on the fence about this one. bfcache will be more highlighted but only in the way it was before.

File: changelog.md (hunk 1 above)
Comment (xoxx761206) [reply changelog.md]: 讚

**Review by xoxx761206 — COMMENTED** [review]
我的網站上使用者

**Pull request merged**